test/functional/p2p_compactblocks.py

```diff
@@ -729,7 +731,7 @@ def test_end_to_end_block_relay(self, listeners):
     # but invalid transactions.
     def test_invalid_tx_in_compactblock(self, test_node, use_segwit=True):
         node = self.nodes[0]
-        assert len(self.utxos)
+        assert_true(len(self.utxos) > 0)
```

Comment:
I make this choice for consistency, across error messages and assert convention. I usually try to avoid missing pure assert and assert that came from utils code like the bitcoin test framework.

But I admit that in this case, it would be better `assert_not_empity(self.utxos)`

However, if you disagree I would restore the previous status with the pure assert